.travis.yml

```diff
@@ -1,50 +1,50 @@
 jobs:
   include:
-    - name: 'Python 3.6'
+    - name: "Python 3.6"
       language: python
-      python: '3.6'
+      python: "3.6"
       sudo: required
       dist: xenial
 
-    - name: 'Python 3.7'
+    - name: "Python 3.7"
       language: python
-      python: '3.7'
+      python: "3.7"
       sudo: required
       dist: xenial
 
-    - name: 'Windows'
+    - name: "Windows"
       language: c # Not really, but travis doesn't support python on Windows
-      python: '3.7'
+      python: "3.7"
       os: windows
 
 install:
-- if [[ "$TRAVIS_OS_NAME" != "windows" ]]; then
-    wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O anaconda.sh;
+  - if [[ "$TRAVIS_OS_NAME" != "windows" ]]; then
+    wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
     export python_version=$TRAVIS_PYTHON_VERSION;
-    bash anaconda.sh -b -p $HOME/anaconda;
-    export PATH="$HOME/anaconda/bin:$PATH";
-  fi
-- if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then
+    bash miniconda.sh -b -p $HOME/miniconda;
```

Comment:
This is a good modification here! Miniconda instead of anaconda might explain how tests are running faster.

Reply:
It was already using miniconda.  It was just renaming it to "anaconda", which was definitely a strange thing to do!